packages/core/src/utils/styleUtils/stylesUtils.ts

```diff
@@ -68,7 +77,7 @@ export const buildCfStyles = ({
   cfColumnSpan,
   cfVisibility,
 }: Partial<StyleProps>): CSSProperties => {
-  return {
+  const translatedRules = {
```

Comment:
Can we agree to call `css` instead of `rules`? We are dealing with css here so in my opinion there is no need to alias it with a different term

Reply:
At this point, it's not yet CSS as keys like `boxSizing` are not valid key names which is why I didn't want to cause confusion by giving it a name that doesn't reflect the variable. How can we call this intermediate state between design value and actual CSS rule that is somehow a mix of both?

Reply:
Ah I just see that React calls it `CSSProperties`! Will rename it accordingly.